run-build.sh

```diff
@@ -60,9 +55,6 @@ while [[ $# > 0 ]]; do
     shift
 done
 
-# Set nuget package cache under the repo
-export NUGET_PACKAGES="$REPOROOT/.nuget/packages"
-
 # Set up the environment to be used for building with clang.
```

Comment:
Looks like this is old news, we shouldn't need this anymore since we don't build any native code in our repo.